.github/workflows/pilot_repo.yml

```diff
@@ -15,6 +15,12 @@ jobs:
         - ppc64le
         - x86_64
     steps:
+        - name: Install QEMU
+          if: env.EESSI_ARCH != 'x86_64'
```

Comment:
This doesn't seem to work, it's still running it on `x86_64` as well. Should it be `matrix`?

Reply:
Fixed in fc132a6, should have been `matrix.EESSI_ARCH` indeed, thanks!